src/core/lender/deployer.sol

```diff
@@ -80,7 +81,7 @@ contract DistributorFab {
     function newDistributor(address currency) public returns (address);
 }
 
-contract SwitchableDistributorFab {
+contract SwitchableFab {
```

Comment:
I think we should keep the pattern.

`ContractName+Fab` therefore it should be `SwitchableDistributorFab`

Reply:
We have this for the Allowance and Whitelist Operators though (the naming pattern)